tests/testsuite/runconfiguration.go

```diff
@@ -0,0 +1,28 @@
+package testsuite
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+var (
+	TestRunConfiguration RunConfiguration
+)
+
+type RunConfiguration struct {
+	WarningToIgnoreList []string
+}
+
+func InitRunConfiguration() {
```

Comment:
Can we use the `originalKV` just to be sure this works if we adjust `AdjustKubeVirtResource`?

Reply:
Thanks! done